shared_model/backend/protobuf/queries/impl/proto_get_account_detail.cpp

```diff
@@ -27,7 +27,23 @@ namespace shared_model {
         : GetAccountDetail(std::move(o.proto_)) {}
 
     const interface::types::AccountIdType &GetAccountDetail::accountId() const {
-      return account_detail_.account_id();
+      return account_detail_.opt_account_id_case()
```

Comment:
I think `account_detail_.has_account_id()` would be more readable

Reply:
```has_account_id()``` is generated as ```private```, so the other variant to check, if it's empty, is to use switch-case, but here it will be awful IMHO